src/utils/file_utils.js

```
@@ -115,9 +115,24 @@ const clear_folder = async (folder_name) => {
   }
 };
 
+const clear_output = async (folder_name, file_name) => {
+  const folder_path = path.join(process.cwd(), folder_name);
+  try {
+    const files = await readdir(folder_path);
+    files.forEach((file) => {
+      if(file==file_name) rm(path.join(folder_path, file), { recursive: true });
+    });
```

We know there can only be one file in the folder with that name, so it's not necessary to check all the file names in the folder. 

You could try to just delete the file and use the `force` [option](https://nodejs.org/docs/latest-v16.x/api/fs.html#fspromisesrmpath-options) to ignore the error in case the file doesn't exist.